src/Equinox.EventStore/EventStore.fs

```diff
@@ -396,7 +396,7 @@ type Context(conn : Connection, batching : BatchingPolicy) =
 
 [<NoComparison; NoEquality; RequireQualifiedAccess>]
 type AccessStrategy<'event,'state> =
-    | EventsAreState
+    | LatestKnownEvent
     | RollingSnapshots of isValid: ('event -> bool) * compact: ('state -> 'event)
```

Comment:
Any renames here for consistency with Cosmos changes? @bartelink

Reply:
isValid and compact should also become `isOrigin` and `toSnapshot`

RollingSnapshots should say:

```
/// Ensures a snapshot is always present (embedded in the stream as an event) every `batchSize` events using the supplied `toSnapshot` function to generate an event from which the state can be reconstituted upon decoding. See https://eventstore.org/docs/event-sourcing-basics/rolling-snapshots/index.html
```

.. or something better